torch_brain/data/sampler.py

```
@@ -488,3 +488,17 @@ def set_epoch(self, epoch: int) -> None:
         """Set the epoch number. Not strictly necessary for sequential sampler
         but included for API compatibility."""
         self.epoch = epoch
+
+
+__all__ = [
+    "RandomFixedWindowSampler",
+    "SequentialFixedWindowSampler",
+    "TrialSampler",
+    "DistributedEvaluationSamplerWrapper",
+    "DistributedStitchingFixedWindowSampler",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)